src/rules/noTripleEqualsNullRule.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * @license
+ * Copyright 2013 Palantir Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as ts from "typescript";
+import * as Lint from "../lint";
+
+const OPTION_NO_NULL_KEYWORD = "no-null-keyword";
```

Comment:
Hmm, this option is confusing, passing this option to the rule seems to make the rule not do anything? Perhaps we should have two options: "allow-null" and "allow-undefined". By default the rule would disallow `=== null`, `!== null`,  `=== undefined`, and `!== undefined`, but if you passed these options it would allow the corresponding constructs.